app/controllers/concerns/course/assessment/question/programming/python/package_concern.rb

```diff
@@ -0,0 +1,128 @@
+# frozen_string_literal: true
```

Comment:
Create a new service for creating programming packages. Put this logic and the supporting template files with the service.